web3/_utils/contract_error_handling.py

```diff
@@ -49,67 +49,47 @@
     "function type.",
 }
 
+MISSING_DATA = "no data"
 
-def raise_contract_logic_error_on_revert(response: RPCResponse) -> RPCResponse:
-    """
-    Reverts contain a `data` attribute with the following layout:
-        "Reverted "
-        Function selector for Error(string): 08c379a (4 bytes)
-        Data offset: 32 (32 bytes)
-        String length (32 bytes)
-        Reason string (padded, use string length from above to get meaningful part)
 
-    See also https://solidity.readthedocs.io/en/v0.6.3/control-structures.html#revert
+def _parse_openethereum_hex_error(data: str) -> str:
```

Comment:
OpenEthereum/Parity are definitely defunct, but I don't know if any other client uses the `Revert`-prefixed `data` format. No docs I've looked through seem to be of any help. Short of testing against all actively-developed clients and seeing what they spit out, I'd say just change naming here, calling it "revert-prefixed" (or something) error data instead of openethereum.